### PR TITLE
Moved content from Announcement to README.md

### PR DESCRIPTION
**Pull request by robbert229**
Just made the readme more descriptive by copy pasting from the original announcement.

### CODE REVIEW

**Comment by robbert229** [comment]
Getting a build error completely unrelated to my changes, and only on mac. Any idea why @victorhurdugaci?

File: README.md [hunk 1]
```diff
@@ -4,5 +4,29 @@ AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/fxhto3omtehio
 
 Travis:   [![Travis](https://travis-ci.org/aspnet/dnx-watch.svg?branch=dev)](https://travis-ci.org/aspnet/dnx-watch)
 
-This project is part of ASP.NET 5. You can find samples, documentation and getting started instructions for ASP.NET 5 at the [Home](https://github.com/aspnet/home) repo.
+With `dnx --watch`, when a file changed, dnx would exit and you had to restart it manually.
```
Comment (Eilon) [inline README.md]: I recommend starting the readme with a brief statement about what the project in this repo does. Something like "Use `dnx-watch` to automatically restart an application when its source code changes."

File: README.md [hunk 2]
```diff
@@ -4,5 +4,29 @@ AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/fxhto3omtehio
...
+`dnx-watch` is a file watcher for `dnx`. It replaces `dnx --watch` and it restarts the process.
+
+### How To Install
+`dnu commands install Microsoft.Dnx.Watcher`
```
Comment (Eilon) [inline README.md]: Should prefix this by saying "From a console window run the following command:"

File: README.md [hunk 3]
```diff
@@ -4,5 +4,29 @@ AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/fxhto3omtehio
...
+
+You might need to pass `--fallbacksource https://myget.org/F/aspnetvnext/api/v3/index.json` if you don't have that feed in the global NuGet config.
```
Comment (Eilon) [inline README.md]: Not exactly - I'd say more accurately that "to install the latest unstable release add the following parameter to the command".

**Comment by robbert229** [comment]
Sorry about the delay, I have implemented your advice. @Eilon

**Comment by Eilon** [comment]
@robbert229 thanks, can you squash the commits into one?

@victorhurdugaci any further comments on this?

**Comment by victorhurdugaci** [comment]
Looks good. I can merge it after @robbert229 will rebase and squash

**Pull request merged**